sections/semantics-forms.include

```diff
@@ -4316,14 +4601,14 @@ part of the form.</p>
   &lt;option value="0"&gt;
   &lt;option value="-30"&gt;
   &lt;option value="30"&gt;
-  <mark class="bad"> &lt;option value="++50"&gt;</mark>
+  <mark class="bad">&lt;option value="++50"&gt;</mark>
 &lt;/datalist&gt;
     </pre>
 
     ...with the following style sheet applied:
 
     <pre highlight="css">
-input { height: 75px; width: 49px; background: #D5CCBB; color: black; }
+input[type="range"] { height: 75px; width: 49px; background: #D5CCBB; color: #000; }
     </pre>
```

Comment:
Why not `<xmp>` on the CSS examples?

Reply:
good call.  didn't think to change it since it didn't have an HTML example in it.  updated.